AutomatedTesting/Assets/TestAnim/scene_export_actor.py

```diff
@@ -208,7 +208,6 @@ def on_update_manifest(args):
 
     global sceneJobHandler
     sceneJobHandler.disconnect()
-    sceneJobHandler = None
```

Comment:
Python also has a ```del``` statement
```
del sceneJobHandler
```

Reply:
I know.  my change removes most of the = None though, so there's no place to put the del...